packages/fxa-settings/src/components/ConnectedServices/Service.tsx

```diff
@@ -0,0 +1,132 @@
+import { ReactComponent as TabletIcon } from './tablet.svg';
+import { ReactComponent as MobileIcon } from './mobile.svg';
+import { ReactComponent as LockwiseIcon } from './lockwise.svg';
+import { ReactComponent as PocketIcon } from './pocket.svg';
+import { ReactComponent as MonitorIcon } from './monitor.svg';
+import { ReactComponent as SyncIcon } from './sync.svg';
+import { ReactComponent as FPNIcon } from './fpn.svg';
+
+export function Service({
+  name,
+  deviceType,
+  location,
+  lastAccessTimeFormatted,
+  canSignOut,
+}: {
+  name: string;
+  deviceType: string;
+  location: DeviceLocation;
+  lastAccessTimeFormatted: string;
+  canSignOut: boolean;
+}) {
+  const { city, stateCode, country } = location;
+  const locationProvided = Boolean(city && stateCode && country);
+  const isPocket = name === 'Pocket';
+  const isMonitor = name === 'Firefox Monitor';
+  const isLockwise = name === 'Firefox Lockwise';
+  const isSync = name === 'Firefox Sync';
+  const isFPN = name === 'Firefox Private Network';
+
+  const renderLink = isPocket || isMonitor || isLockwise || isSync || isFPN;
+  let serviceLink = '';
+  let iconSvg = <WebIcon data-testid="web-icon" />;
+
+  if (renderLink) {
+    if (isPocket) {
+      serviceLink = 'https://www.mozilla.org/en-US/firefox/pocket/';
+      iconSvg = <PocketIcon data-testid="pocket-icon" />;
+    }
+    if (isMonitor) {
+      serviceLink = 'https://monitor.firefox.com/';
+      iconSvg = <MonitorIcon data-testid="monitor-icon" />;
+    }
+    if (isLockwise) {
+      serviceLink = 'https://www.mozilla.org/en-US/firefox/lockwise/';
+      iconSvg = <LockwiseIcon data-testid="lockwise-icon" />;
+    }
+    if (isSync) {
+      serviceLink =
+        'https://support.mozilla.org/en-US/kb/how-do-i-set-sync-my-computer';
+      iconSvg = <SyncIcon data-testid="sync-icon" />;
+    }
+    if (isFPN) {
+      serviceLink = 'https://vpn.mozilla.com/';
+      iconSvg = <FPNIcon data-testid="fpn-icon" />;
+    }
+  } else {
+    const lowerName = name.toLowerCase();
+    if (deviceType === 'mobile') {
+      iconSvg = <MobileIcon data-testid="mobile-icon" />;
+    }
+    if (lowerName.includes('ipad')) {
+      iconSvg = <TabletIcon data-testid="tablet-icon" />;
+    }
+    if (deviceType === 'desktop') {
+      iconSvg = <DesktopIcon data-testid="desktop-icon" />;
+    }
+  }
+
+  return (
+    <div className="my-1" id="service" data-testid="settings-connected-service">
+      <div className="p-4 border-2 border-solid border-grey-100 rounded flex mobileLandscape:justify-around items-center flex-col mobileLandscape:flex-row">
+        <div className="flex flex-grow w-full mobileLandscape:flex-2">
+          <span className="flex justify-center items-center flex-1">
+            {iconSvg}
+          </span>
+          <div className="flex flex-col flex-5 mobileLandscape:flex-row">
+            <div className="flex flex-col mobileLandscape:flex-2">
+              {renderLink ? (
```

Comment:
seems simpler to get rid of `renderLink` and just do `if (!!name)` here

Reply:
There will always be a name, we handle service's with links differently here. In the case that it's just a regular device we only render the name (no link)